resource/template/resource.go

```diff
@@ -88,10 +88,6 @@ func NewTemplateResource(path string, config Config) (*TemplateResource, error)
 		tr.Prefix = config.Prefix
 	}
 
-	if !strings.HasPrefix(tr.Prefix, "/") {
```

Comment:
This might break other backends, we should add some tests that cover this behavior or change etcdv3 backend explicitly.